schemas/put-artifact-response.yml

```diff
@@ -0,0 +1,13 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Put Artifact Response"
+description: complete a blob artifact upload
+type:               object
+properties:
+  outcome:
+    type: string
+    enum:
+      - success
+      - failure
```

Comment:
Because we don't change error codes when you use reply.... Communicate errors with error responses... Please... 

Otherwise tc-client won't throw an exception as expected..